include/tscore/HTTPVersion.h

```diff
@@ -40,7 +40,7 @@ class HTTPVersion
 
   uint8_t get_major() const;
   uint8_t get_minor() const;
-  int get_flat_version() const;
+  uint32_t get_flat_version() const;
```

Comment:
Is this really part of your PR?

Reply:
Yes, it coincides with the fix for the warning generated (and fixed in this PR) here:

https://github.com/apache/trafficserver/pull/7829/files#diff-7f286e5169bc2d28c13f1ceaf3bb482b36cdb0743fbac4040fd4cd5a9c0897c1R4034

`%x` takes an unsigned integer, so I changed this to an unsigned return.